.travis.yml

```diff
@@ -105,9 +105,25 @@ install:
 script:
   - echo $TYPE $DISTRO $MONOLITHIC $SYSTEMD $WERROR
   - set -e
-  - if [ -n "$LINT" ] ; then python3 -t -t -E -W error testing/check_fc_files.py ; fi
   - make bare
   - make conf
+  - |
+      if [ -n "$LINT" ] ; then
+        # Run filecontext checker
+        python3 -t -t -E -W error testing/check_fc_files.py
+
+        # Install SELint from Debian testing
+        wget -O - https://ftp-master.debian.org/keys/archive-key-10.asc 2>/dev/null | sudo apt-key add -
+        sudo add-apt-repository 'deb http://deb.debian.org/debian/ testing main' -y
+        sudo apt-get update -q
+        sudo apt-get install -y selint
+
+        # Run SELint
+        selint -V
+        selint --source --recursive --summary --fail --disable C-005 --disable S-010 --disable W-005 .
```

Comment:
Why not S-010?  Is that just a temporary thing until the issues it reports are cleaned up?

Reply:
There are currently 96 findings of this tag.
So I disabled it for now.